### PR TITLE
remove-forgotten-reviewer

### PR DESCRIPTION
**Pull request by blankdots**

### CODE REVIEW

**Review by jbygdell — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -11,7 +11,6 @@ updates:
       - "pontus"
       - "jonandernovella"
```
Comment (jbygdell) [inline .github/dependabot.yml:12]:
```suggestion
```

**Review by jbygdell — COMMENTED** [review]
Might as well remove 2 others at the same time.

**Comment by blankdots** [comment]
seems our pipeline is broke for some reason :thinking:

**Review by norling — APPROVED** [review]
Looks very reasonable!

**Review by viklund — APPROVED** [review]

**Pull request merged**